.circleci/config.yml

```diff
@@ -84,9 +84,9 @@ executors:
           MB_DB_DBNAME: circle_test
           MB_DB_USER: root
           MB_MYSQL_TEST_USER: root
-      - image: circleci/mysql:latest
+      - image: cimg/mysql:8.0
```

Comment:
Same. I know pinning to a specific version is nice for CI not breaking without warning, but in this case I think it's a feature rather than a bug since find out the new version is broken right away rather than waiting for a customer to tell us. We're officially committed to supporting the latest stable versions of these DBs so we should test against them